packages/office-ui-fabric-react/src/components/Button/Button.types.ts

```diff
@@ -223,6 +223,12 @@ export interface IButtonProps extends React.AllHTMLAttributes<HTMLAnchorElement
   getSplitButtonClassNames?: (disabled: boolean,
     expanded: boolean,
     checked: boolean) => ISplitButtonClassNames;
+
+  /**
+  * Defines a custom key that opens button menu. If the value is not provided, down arrow key is used.
+  * If set to null, this will be disabled.
```

Comment:
Provides a custom KeyCode that can be used to open the button menu. The default keyCode is the down arrow. A value of null can be provided to disable the key codes for opening the button menu.